crt_portal/cts_forms/templates/forms/complaint_view/index/filters/checkbox_section_filter.html

```diff
@@ -4,12 +4,17 @@
 <form id="cts-forms-profile" class="usa-form" method="post" action="{% url 'crt_forms:cts-forms-profile' %}" novalidate>
   {% csrf_token %}
   <div class="crt-dropdown" data-crt-dropdown>
-    <button class="crt-dropdown__title crt-section_dropdown" aria-controls="assigned-section" aria-expanded="true"
-      type="button" data-crt-dropdown-control>
+    <button
+      class="crt-dropdown__title crt-section_dropdown"
+      aria-controls="assigned-section"
+      aria-expanded="false"
+      type="button"
+      data-crt-dropdown-control
+    >
       <span class="label">{{ form.get_section_filters|multiselect_summary:"View sections" }}</span>
       <span alt="open filter dropdown" class="icon" />
     </button>
-    <div class="content" id="assigned-section">
+    <div class="content" id="assigned-section" hidden>
```

Comment:
Just curious, how come this is being hidden?

Reply:
It's being hidden because there's a script (https://github.com/usdoj-crt/crt-portal/blob/develop/crt_portal/static/js/dropdown.js#L40) that applies the hidden attribute on page load. But because it waits for the page to load, it's what causes the dropdowns to all flash "visible" briefly before the JS hides it. This was really bugging me, so the fix is to have it rendered as hidden by default instead.

Reply:
Yessssssssss!!!!!!!!!!!

Thank you!